packages/harmony/.svgrrc.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
   titleProp: true,
   descProp: true,
   replaceAttrValues: {
-    red: '{props.fillColor}',
-    '#FF0000': '{props.fillColor}'
+    red: '{props.fillcolor}',
```

Comment:
Heads up I fix this in an open pr, mind dropping?

Reply:
yea